cmd/kubeadm/app/constants/constants.go

```diff
@@ -56,4 +56,18 @@ const (
 	// Minimum amount of nodes the Service subnet should allow.
 	// We need at least ten, because the DNS service is always at the tenth cluster clusterIP
 	MinimumAddressesInServiceSubnet = 10
+
+	// DefaultTokenDuration specifies the default amount of time that a bootstrap token will be valid
+	DefaultTokenDuration = time.Duration(8) * time.Hour
```

Comment:
Yes, I'm gonna change it to `0` later, just wanna stay away from functional changes here